manifests/c/CONZEInformatik/Vistrax/3.3.23061.1234/CONZEInformatik.Vistrax.installer.yaml

```
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.2.10 $debug=NVS0.CRLF.5-1-22621-2428.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: CONZEInformatik.Vistrax
+PackageVersion: 3.3.23061.1234
+InstallerLocale: en-US
+InstallerType: zip
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /quiet
+  SilentWithProgress: /passive
+UpgradeBehavior: install
+ReleaseDate: 2023-03-02
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.DotNet.Framework.DeveloperPack_4
+Installers:
+- Architecture: x86
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: vistrax_setup_3.3.23061.1234.exe
+  InstallerUrl: https://update.vistrax.de/fileadmin/downloads/vistrax_setup_3.3.23061.1234.zip
+  InstallerSha256: 4960CD21818469D8136F789698B9B407071790B78625071B56388116E77E9BE4
+  InstallerSuccessCodes: '3010'
```

```suggestion
  InstallerSuccessCodes:
  - 3010
```
@FaserF Sorry for the multiple changes. I've copied this from another PR that was accepted, and updated the integer. Hopefully this time will work.